radix-engine/src/errors.rs

```diff
@@ -271,13 +276,23 @@ pub enum CreateObjectError {
 
 #[derive(Debug, Clone, PartialEq, Eq, ScryptoSbor)]
 pub enum SystemModuleError {
+    BlueprintSchemaValidationError(BlueprintSchemaValidationError),
```

Comment:
Can we move `BlueprintSchemaValidationError` to `SystemError` as there is no payload validation system module?